docs/statusbar.md

```diff
@@ -9,14 +9,22 @@ Component to control the app status bar.
 
 It is possible to have multiple `StatusBar` components mounted at the same time. The props will be merged in the order the `StatusBar` components were mounted.
 
-```jsx
-<View>
-  <StatusBar backgroundColor="blue" barStyle="light-content" />
-  <View>
-    <StatusBar hidden={route.statusBarHidden} />
-    ...
-  </View>
-</View>
+```SnackPlayer name=rn-statusbar
```

Comment:
I think that `StatusBar` will be a friendlier name 😉 

Also `web` should be disabled using `supportedPlatforms` parameter since `StatusBar` has no visual representation on that platform.